torch/_inductor/runtime/triton_helpers.py

```diff
@@ -198,25 +198,38 @@ def any(a, dim):
 
 @triton.jit
 def bucketize_binary_search(
-    values,  # 1D tensor
-    offsets_ptr,
-    indexing_dtype,
+    values: tl.tensor,
+    offsets_ptr: tl.pointer_type,
+    sorter_ptr: tl.pointer_type,  # optional pointer of the same shape as offsets_ptr, None if not present
```

Comment:
tbh never seen tl.pointer_type - but should this be Optional[tl.pointer_type] ?

Reply:
@davidberard98 I tried annotating it with that, but Triton apparently parses the type annotations and barfed on `Optional`, so I figured commenting about it was the next best thing.